include/xtensor/xstrided_view.hpp

```diff
@@ -108,7 +108,7 @@ namespace xt
      *
      * @sa strided_view, transpose
      */
-    template <class CT, class S, layout_type L = layout_type::dynamic, class FST = detail::flat_storage_getter<CT>>
+    template <class CT, class S, layout_type L = layout_type::dynamic, class FST = detail::flat_storage_getter<CT, L>>
```

Comment:
Ifem, should we use `flat_storage_getter<CT, XTENSOR_DEFAUL_TRAVERSAL>`, and not need the `order` trait.